scripts/qa_crowbarsetup.sh

```diff
@@ -1613,6 +1613,7 @@ function onadmin_wait_tftpd
 {
     wait_for 300 2 \
         "timeout -k 2 2 tftp $adminip 69 -c get /discovery/x86_64/bios/pxelinux.cfg/default /tmp/default"
+        "tftpd server ready"
```

Comment:
needs to be in the same statement, so either drop the linebreak or add a backslash `\`